tests/part_test.py

```diff
@@ -661,6 +662,13 @@ def test_get_disk_parts_empty(self):
         with self.assertRaises(GLib.GError):
             BlockDev.part_get_disk_parts (self.loop_dev)
 
+
+def _round_up_mib(size):
+    # convert size to nearest MiB (up)
+    rounded = Size(size).round_to_nearest(Size(1024**2), rounding=ROUND_UP)
+    return int(rounded.convert_to(B))
```

Comment:
You can do `rounded.get_bytes()` or even just `int(rounded)`.

Reply:
Fixed.